exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -196,9 +197,10 @@ CREATE TABLE IF NOT EXISTS %s %s (
      INDEX idx_span_attr_value mapValues(SpanAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
      INDEX idx_duration Duration TYPE minmax GRANULARITY 1
 ) ENGINE = %s
+PARTITION BY toDate(TimestampTime)
+PRIMARY KEY (ServiceName, SpanName, TimestampTime)
```

Comment:
In the latest revision I have removed this column in favor of simply using the `Timestamp` converted to `DateTime`. I hope to make a similar change for logs because having a separate time column is inconvenient